gtda/pipeline.py

```diff
@@ -522,6 +522,6 @@ def make_pipeline(*steps, **kwargs):
     """
     memory = kwargs.pop('memory', None)
     if kwargs:
-        raise TypeError('Unknown keyword arguments: "{}"'
-                        .format(list(kwargs.keys())[0]))
+        raise TypeError(
+            f'Unknown keyword arguments: "{list(kwargs.keys())[0]}"')
```

Comment:
Btw, why `list(...)[0]` ?

Reply:
`TypeError: 'dict_keys' object is not subscriptable`

Reply:
Yes, sorry. I was referring to the`[0]` part- why only the first element? I mean, don't you want to print all arguments?

Reply:
Not sure. I think this one was copied from `scikit-learn`...

Reply:
https://github.com/scikit-learn/scikit-learn/blob/b194674c42d54b26137a456c510c5fdba1ba23e0/sklearn/pipeline.py#L702